nalito223/todos
# Feature/dnd

### Summary:

This PR adds drag and drop functionality and the basic structure of the todos board. 

_______________
### How to test:

Note: Form and buttons are not yet active. 

- Clone [API](https://github.com/nalito223/api-todos) and [Source code ](https://github.com/nalito223/todos/)
- Run the corresponding commands:
    - API: `nodemon server.js`
    - Source code: `npm start `
- See that tasks match the gif below. (Dismiss typescript error)
- See that UI matches the gif below
- Test dragging and dropping the tasks into different columns and orders. See that the changes are preserved when the task is dropped.

____________
![prdnd](https://user-images.githubusercontent.com/110144802/212147274-809565e9-53b5-4401-8252-b2fdd3141aae.gif)
